prompts/power-apps/event-and-rsvp-manager/assets/sample.json

```diff
@@ -36,9 +36,9 @@
     ],
     "authors": [
       {
-        "gitHubAccount": "summitbaj",
-        "name": "Summit Bajracharya",
-        "pictureUrl": "https://github.com/summitbaj.png"
+        "gitHubAccount": "mmbr1606",

```

Comment:
You're changing the sample.json prompt that someone else submitted to your details. That doesn't make any sense. Can you please change this back to the original value?